native/packages/react-native-bpk-component-switch/readme.md

```diff
@@ -0,0 +1,48 @@
+# react-native-bpk-component-switch
+
+> Backpack React Native switch component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-switch --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import BpkSwitch from 'react-native-bpk-component-switch';
+import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: TOKENS.spacingBase,
+  }
+});
+
+export default class App extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <BpkSwitch />
+        <BpkSwitch value />
```

Comment:
I don't like `value` for the prop. Should be `on` IMO

Reply:
Neither do I, but that's the same prop the underlying component uses and I figured it's nicer to use the same as them. Happy to change though if enough people disagree. Any thoughts @matthewdavidson @matteo-hertel?

Reply:
If the underlying component is using `value` I'd stick with it, we'll not add much value to the end user by changing it and this component will become an opinionated wrapper of the native switch

Reply:
Hmm. It just looks extremely strange when using the shorthand with this version, but maybe that's not common

Reply:
I know what you mean, it does look weird! If it were `<BpkSwitch value={true} />` it might look less weird, but `eslint` corrects it to just `value`.

Reply:
Stick with `value` 👍